lib/restrictedsession/restricted_test.go

```diff
@@ -152,20 +149,32 @@ func setupBPFContext(t *testing.T) *bpfContext {
 		t.Skip("Tests for package restrictedsession can only be run as root.")
 	}
 
+	// Create temporary directory where cgroup2 hierarchy will be mounted.
+	// DO NOT MOVE THIS LINE.
+	// t.TempDir() creates t.Cleanup() action. If this hook is called before
+	// we mount cgroup in Close() this test will fail.
+	cgroupDir := t.TempDir()
+
+	tt := bpfContext{
```

Comment:
```suggestion
	bpfCtx := bpfContext{
```

`tt` is typically a test case, which this doesn't seem to be.